vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteSpaceLinkController.java

```diff
@@ -13,7 +13,7 @@
 import edu.asu.diging.vspace.core.services.ISpaceLinkManager;
 
 @Controller
-public class DeleteLinkController {
+public class DeleteSpaceLinkController {
```

Comment:
What do you mean? I renamed it `DeleteSpaceLinkController` because it was housing methods for deleting module slide links and space links and you didn't want to mix them. So now there's `DeleteSpaceLinkController` and `DeleteSlideLinkController`

Reply:
do you want me to just keep the name as `DeleteLinkController`?

Reply:
yes, this class does not just delete spacelinks, so the name is misleading.